setup.py

```diff
@@ -0,0 +1,46 @@
+import setuptools
+import os
+
+
+def read_requirements():
+    build_dir = os.path.abspath(os.path.dirname(__file__))
+    with open(os.path.join(build_dir, "yolact", "requirements.txt")) as f:
+        return f.read().splitlines()
+
+
+def packages():
+    return [
+        'yolact',
```

Comment:
not an issue currently, but noting that nested directories/modules will not be picked up

Reply:
Addressed!